src/components/Auth.vue

```diff
@@ -486,10 +487,10 @@ export default {
     },
     changeSemester: function (year) {
       const currentMonth = new Date().getMonth();
-      const sem = currentMonth >= 4 && currentMonth <= 10
+      const sem = currentMonth >= 5 && currentMonth <= 10
```

Comment:
I think this is fine for the quick fix but we should also apply this same logic to the other bound; right now in November it will switch to Spring (or maybe IAP I forget).

Reply:
I think that's fine because class year doesn't change then, right? So now it's a question of what's what for highlighting purposes, and I don't think that's important enough to warrant a manual adjustment every time this number goes in or out of FireRoad like we have to for the May switch